SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Distribute.cs

```diff
@@ -51,6 +51,30 @@ static void HandleUpdateAction(UpdateAction updateAction)
             }
         }
 
+        static void SetUpdateTrack(UpdateTrack updateTrack)
+        {
+            switch (updateTrack)
+            {
+                case UpdateTrack.Public:
+                    AndroidDistribute.UpdateTrack = 1;
+                    break;
+
+                case UpdateTrack.Private:
+                    AndroidDistribute.UpdateTrack = 2;
+                    break;
+            }
+        }
+
+        static UpdateTrack GetUpdateTrack()
+        {
+            var updateTrackValue = AndroidDistribute.UpdateTrack;
+            if (updateTrackValue == 2)
```

Comment:
In C#, you can [cast the integer to enum directly](https://docs.microsoft.com/en-us/dotnet/api/system.enum?view=netframework-4.8#performing-conversions):
```csharp
(UpdateTrack)updateTrackValue
```